src/lint/lint-ts.ts

```diff
@@ -1,41 +1,84 @@
-import { Linter, LintResult } from "tslint";
+import { Linter, LintResult, Configuration } from "tslint";
+import { resolve } from "path";
+import { CommandModule, Argv, argv } from "yargs";
 
 import { Logger } from "../utils/logger";

```

Comment:
import using the `../utils/index`? or was there a reason not too?

same for the below